english/bake-cookies/cookies.md

```diff
@@ -1,8 +1,9 @@
-Baking is therapeutic for you.  You remember
-learning to bake from your grandmother and how
+Baking is therapeutic for you. Smell that radiates 
+through out the house, gives us all that warm fuzzy feeling.
+You remember learning to bake from your grandmother and how
 relaxing and happy that always makes you.  You
 want to spread the joy (and chocolate chip cookies).
-Probably you love cookies too.
+Probably you love cookies too. 
```

Comment:
Looks like you added an extra space at the end of this line. Your other modification looks good, so if you remove this space, I'll be happy to merge this!